### DISCUSSION TITLE
Bokeh 3.0 Announcement Blog Post

### DISCUSSION THREAD

**Discussion by vicadesoba** [bokeh/bokeh#12310] [General]
In preparation for the 3.0 release, below are the items I plan to spotlight in the announcement blog post. If you have any suggestions on items that should be added or removed from the list (or general feedback/content you feel would be good to mention/include), feel free let me know.

- https://github.com/bokeh/bokeh/pull/12211
- https://github.com/bokeh/bokeh/discussions/12021/ (https://github.com/bokeh/bokeh/pull/12020)
- https://github.com/bokeh/bokeh/issues/12288
- https://github.com/bokeh/bokeh/pull/12294
- https://github.com/bokeh/bokeh/pull/12302

**Comment by ianthomas23** [comment]
Also #11914.

**Reply by vicadesoba** [reply]
Ah yes, thanks @ianthomas23!
